resources/views/layouts/default/downloads/table.blade.php

```diff
@@ -1,7 +1,7 @@
 <ul class="list-group">
   @foreach($files as $file)
     <li class="list-group-item">
-      <a href="{{route('frontend.downloads.download', [$file->id])}}" target="_blank">
+      <a href="{{route('frontend.downloads.download', [$file->id])}}" target="_blank" @if(is_null($file->disk) && !str_contains($file->url, config('app.url'))) data-external-redirect="{{ $file->url }}" @endif>
```

Comment:
I think it's better to add a property on the model to check this, especially since it's in a front-end template, and if we need to change the condition for whatever reason

Reply:
So, if I understand correctly, you want me to add an attribute to the model, like `isExternalFile`, with a getter that returns the boolean value?

Reply:
Exactly, I understood the same... An attribute at model, a simple check in blade.